src/styles/main.scss

```diff
@@ -1 +1,66 @@
 // put code here
+$watch-size: 80vmin;
+
+.watch {
+  height: $watch-size;
+  width: $watch-size;
+  border-radius: 50%;
+  border: 5px dashed black;
+
+  &__arrows {
+    position: relative;
+    width: 100%;
+    height: 100%;
+  }
+
+  &__seconds {
+    position: absolute;
+    left: #{$watch-size / 2 - $watch-size / 112};
+    width: #{$watch-size / 56};
+    height: #{$watch-size * 0.5};
+    background-color: green;
+    transform-origin: bottom center;
+    animation-iteration-count: infinite;
+    animation-timing-function: linear;
+    animation-duration: 5s;
+    animation-name: rotating;
+  }
+
+  &__minutes {
+    position: absolute;
+    top: #{$watch-size / 2 - $watch-size * 0.25};
+    left: #{$watch-size / 2 - $watch-size / 48};
```

Comment:
![image](https://github.com/mate-academy/layout_stop-watch/assets/96409563/e6933a42-e8bc-48e9-aa4e-5a8e5f1c408f)
Instead of hardcoding the exact sizes and distances, you can just center them with `transform: translate`
https://www.w3schools.com/css/tryit.asp?filename=trycss_align_transform